ably/options_test.go

```diff
@@ -1,57 +1,44 @@
 package ably_test
 
 import (
-	"log"
 	"net/url"
 
 	"github.com/ably/ably-go/ably"
 
 	. "github.com/ably/ably-go/Godeps/_workspace/src/github.com/onsi/ginkgo"
 	. "github.com/ably/ably-go/Godeps/_workspace/src/github.com/onsi/gomega"
-	"github.com/ably/ably-go/Godeps/_workspace/src/github.com/onsi/gomega/gbytes"
 )
 
 var _ = Describe("ClientOptions", func() {
-	var (
-		options *ably.ClientOptions
-		buffer  *gbytes.Buffer
-	)
+	var opts *ably.ClientOptions
 
 	BeforeEach(func() {
-		buffer = gbytes.NewBuffer()
+		opts = &ably.ClientOptions{}
+	})
 
-		options = &ably.ClientOptions{
-			Key: "id:secret",
-		}
+	Context("when Key is valid", func() {
+		var err error
 
-		options.Prepare()
-	})
+		BeforeEach(func() {
+			err = opts.SetKey("name:secret")
```

Comment:
Dependant fields problem, if you can tell me how to solve that differently or that we don't care, then I'll change that.